native/src/modules/endpoint/reducers/morphContentLanguage.js

```diff
@@ -156,34 +164,39 @@ const poiRouteTranslator = (newPois: $ReadOnlyArray<PoiModel>, newLanguage: stri
     }
   }
 
+const translateRoutes = (state: CityContentStateType, action: MorphContentLanguageActionType): RouteMappingType => {
+  const { routeMapping, city } = state
+  const { newCategoriesMap, newEvents, newPois, newLanguage } = action.params
+
+  const categoryTranslator = categoryRouteTranslator(newCategoriesMap, city, newLanguage)
+  const eventTranslator = eventRouteTranslator(newEvents, newLanguage)
+  const poiTranslator = poiRouteTranslator(newPois, newLanguage)
+
+  return mapValues(routeMapping, route => {
+    if (route.routeType === 'category') {
```

Comment:
Do not like the string-comparison. Wouldn't enum or constant be nicer?

Reply:
I think flow warns if you type: `if (route.routeType === 'catttttegory')` At least typescript does this.

Reply:
routeType already is kind of an enum as it only can have `category | event | poi` as value.